### PR TITLE
Refactor dependencies

### LINKED ISSUE #111: Refactor gradle dependencies
## Overview (Required)
I prefer creating the versions file and use the variables like this.
https://github.com/konifar/material-cat/blob/introduce_dagger/versions.gradle
https://github.com/konifar/material-cat/blob/introduce_dagger/app/build.gradle#L120

I think it's good way if we'll divide our app to some modules.
What do you think?

### PR DESCRIPTION
**Pull request by konifar**
## Issue
- close #111

## Overview (Required)
- This is proposal for refactoring gradle dependencies.
- Create versions.gradle to manage all libraries version.
- This makes dependencies in build.gradle simple.

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: app/build.gradle [hunk 1]
```diff
@@ -66,117 +66,92 @@ android {
...
-    // Firebase
-    // Adjust version for emulator
-    def firebaseVersion = '11.6.2'
-    implementation "com.google.firebase:firebase-firestore:$firebaseVersion"
-    implementation "com.google.firebase:firebase-auth:$firebaseVersion"
-    implementation "com.google.firebase:firebase-core:$firebaseVersion"
-    implementation('com.crashlytics.sdk.android:crashlytics:2.7.1@aar') {
+    implementation depends.kotlin.stdlib
+
+    //==================== Support Library ====================
```
Comment (takahirom) [inline app/build.gradle]: 👍

**Comment by takahirom** [comment]
Please fix this 🙇 

```
:app:compileDebugKotline: /home/circleci/project/app/src/debug/java/io/github/droidkaigi/confsched2018/presentation/DebugApp.kt: (4, 21): Unresolved reference: leakcanary
e: /home/circleci/project/app/src/debug/java/io/github/droidkaigi/confsched2018/presentation/DebugApp.kt: (23, 9): Unresolved reference: LeakCanary
 FAILED

FAILURE: Build failed with an exception.
```

**Comment by konifar** [comment]
Fixed!
@takahirom Can you review again? 🙏

**Comment by takahirom** [comment]
👍 💯

**Pull request merged**